protocol/__init__.py

```diff
@@ -118,13 +122,15 @@ def register_host(self):
         Register a host as the datatrust in protocol
         """
         log.info('****Registering host****')
+        tx_count = self.w3.eth.getTransactionCount(self.datatrust_wallet) + 1
```

Comment:
what is this for?

Reply:
Ah, leftover from when I was troubleshooting and manually specifying the nonce.

Reply:
Removed